stlab/future.hpp

```diff
@@ -1042,24 +1047,33 @@ namespace detail
             , _holds(_remaining)
         {}
 
-        std::atomic_size_t                    _remaining{0};
+        std::atomic_size_t                    _remaining;
```

Comment:
The value is anyway initialized in the c'tor with the necessary number of loops.